Userland/Utilities/fortune.cpp

```diff
@@ -46,6 +46,18 @@ class Quote {
     ByteString const& url() const { return m_url; }
     Optional<ByteString> const& context() const { return m_context; }
 
+    size_t length() const
+    {
+        // TODO: Should we consider m_utc_time's length as well?
+        size_t total = 0;
+
+        total = m_quote.length() + m_author.length() + m_url.length();
+        if (m_context.has_value())
+            total += m_context.value().length();
```

Comment:
I _think_ `m_context->length()` should work, but I can't confirm it right now. It's also a very minor nitpick.

Reply:
I tried that and it didn't work. Crashed when `m_context` didn't have a value.

Reply:
The `.has_value()` check still needs to be there, yes, but the very verbose `.value().` can be replaced with `->`.